static/src/javascripts/projects/common/views/contributions-embed.html

```diff
@@ -1,3 +1,39 @@
-<figure class="element element-interactive contribute-embed interactive element--inline" data-interactive="https://interactive.guim.co.uk/embed/iframe-wrapper/0.1/boot.js" data-canonical-url="https://interactive.guim.co.uk/contributions-embeds/embed/embed.html" data-alt="Contribute">
-    <a id="js-giraffe__contribute-button" href="<%=linkHref%>" data-link-name="in body link" class="u-underline"></a>
+<figure class="element element-interactive interactive element--<%=position%> contribution-embed <%=bottomClass%>">
+    <div class="interactive-embed">
+        <input type="hidden" id="hidden-value" name="amount" class="js-amount js-amount-hidden" value=50>
+        <div class="flex-container">
+            <div >
+                <h3 class="form__column-heading title">Support our journalism</h3>
+            </div>
+
+            <div >
+                <p class="form-section">If you use it, if you like it, why not pay for it? It's only fair. Please give to the Guardian.</p>
+                <p class="form-section">Make a one-off contribution:</p>
+            </div>
+            <div class="amount-field <%=bottomClass%>">
+                <div>
+                    <div class="js-button-group button-group u-cf">
+                        <button type="button" tabindex="8" class="js-button option-button option-button--bold <%=bottomClass%>" data-amount=25>
```

Comment:
Ok cool. I was just checking for future reference whether this causes any problems! Will change to keep the style.